src/OpenTelemetry.Exporter.InfluxDB/CHANGELOG.md

```diff
@@ -1 +1,5 @@
 # Changelog
+
+## 1.0.0-alpha.1
+
+Released 2023-May-18
```

Comment:
Add some text under this. Refer to [this](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.Runtime/README.md) for example.

Reply:
Fixed with #1197.